web/styles/settings.css

```diff
@@ -1790,7 +1778,7 @@ label.preferences-radio-choice-label {
 
 /* This value needs to match with the same in subscriptions.css, as
```

Comment:
Should this comment be removed or updated, given the movement of styles out of the `subscriptions.css` file?

Reply:
I don't think so, since this is still present in `subscriptions.css`

```
/* Note that this block has settings_page CSS as well, and thus needs
   to match the media queries in settings.css.

   Longer-term we should extract this logic two-column-overlay class
   to read more naturally. */
```